website/docs/components/app-side-nav/partials/code/how-to-use.md

```diff
@@ -175,7 +175,7 @@ When the App Side Nav is used in conjunction with portals, the nesting of naviga
 As mentioned above, the full-fledged `Hds::AppSideNav` component is “responsive” by default:
 
 - when the **viewport is `desktop`**, the sidebar navigation is static and has a fixed width
-    - the width at which the viewport is considered “desktop” is controlled by a dedicated CSS variable `--hds-app-desktop-breakpoint`; if needed it can be overwritten (at `:root` level) to define a custom “desktop” breakpoint
+    - the width at which the viewport is considered “desktop” can be overwritten using the `@breakpoint` argument
```

Comment:
Note: also need to add `@breakpoint` to the AppSideNav component API section

Reply:
I was sure I've added it. Clearly not 🤦‍♂️